templates/Page.ss

```diff
@@ -26,7 +26,7 @@ Change it, enhance it and most importantly enjoy it!
 	<% require themedCSS('typography') %>
 	<% require themedCSS('form') %>
 	<% require themedCSS('layout') %>
-	<link rel="shortcut icon" href="$ThemeDir/images/favicon.ico" />
+	<link rel="shortcut icon" href="/themes/simple/images/favicon.ico" />
```

Comment:
Please use non-root relative links for this. i.e. `href="themes/simple/images/favicon.ico"` not `href="/themes/simple/images/favicon.ico"`, as the leading `/` will break out of base_dir